services/provider/server/server.go

```diff
@@ -74,6 +77,12 @@ const (
 	clientInfoRbdClientProfileKey    = "csiop-rbd-client-profile"
 )
 
+var (
+	lastKnownInstalledCSVname string
+	lastKnownSubChannelName   string
+	lastKnownMutex            sync.Mutex
```

Comment:
The name does not make sense, the mutex is not "last known", may I offer an alternative name:
```suggestion
	subChannelReadMutex            sync.Mutex
```

Reply:
Used a new name